types/new-relic-browser/new-relic-browser-tests.ts

```diff
@@ -1,3 +1,5 @@
+import * as newrelic from "new-relic-browser";
```

Comment:
Since this now uses a commonjs-style `export =`, the import should match: `import newrelic = require("new-relic-browser")`